File: src/ex01/ex01.java
package ex01;

public class ex01 {
    public static void main(String[] args){
        int conv = 0;
        int a = 10;
        int b = 20;

        conv = a;
        a = b;
        b = conv;

        System.out.println("A variável a tem valor: "+ a);
        System.out.println("A variável a tem valor: "+ b);
    }
}

File: ExC#/ex-01.cs
using System;

public class Ex01
{
    public static void Main(string[] args)
    {
        int a = 10;
        int b = 20;
        int c = 0;
        
        c = a;
        a = b;
        b = c;
        
        Console.WriteLine("Var a, b");
        Console.WriteLine(a);
        Console.WriteLine(b);
        
    }
}